spacy/lang/pt/__init__.py

```diff
@@ -17,13 +18,19 @@
 class PortugueseDefaults(Language.Defaults):
     lex_attr_getters = dict(Language.Defaults.lex_attr_getters)
     lex_attr_getters[LANG] = lambda text: 'pt'
+    BASE_NORMS.update({
```

Comment:
Instead of updating the global `BASE_NORMS` here, we should probably keep this separate. Maybe even in a `norm_exceptions.py`, to keep things consistent with the other languages.

`add_lookups` can take a variable number of arguments, so we could do something like this:

```python
from .norm_exceptions import NORM_EXCEPTIONS

lex_attr_getters[NORM] = add_lookups(Language.Defaults.lex_attr_getters[NORM], 
                                     BASE_NORMS, NORM_EXCEPTIONS)
```